### ISSUE TITLE
Expansion scenarios for deep convection at SGP

### ISSUE THREAD

**Issue by wgustafson** [ARM-DOE/lasso-public#3]
This thread is for discussing opportunities to expand LASSO to do simulations of deep convection at ARM's [Southern Great Plains (SGP)](https://www.arm.gov/capabilities/observatories/sgp) location in Oklahoma. Some background information can be found in the [associated blog entry](https://www.arm.gov/news/blog/post/52252) . Additional discussion threads will be created as the [LASSO Expansion blog series](https://www.arm.gov/news-events/search?news-category%5B0%5D%5B0%5D=post&news-category%5B1%5D%5B0%5D=LASSO) focuses on additional sites.

How would you configure routine simulations of deep convection for ARM? How would you prioritize in relation to other LASSO expansion scenarios, such as those discussed in previous posts in this series?  What observations would you want included in the data bundle? What new observations would ARM need to add to complement and/or evaluate the LES? What science questions would you like the simulations to support?

Note ARM's [Comment Policy](https://github.com/ARM-DOE/lasso-public/wiki/Comment-Policy).

**Comment by eroesler** [comment]
For the MOSAiC field campaign, I would like to pursue a location-adaptive grid nested in a larger regional model either online or offline.  Uninformed question here, but would deep convection be needed for MOSAiC?

**Comment by wgustafson** [comment]
See my response in #2 for the MOSAiC convection. Note that I changed the title for #3--I forgot to hit the "save" button in this issue after modifying text I copied from the MOSAiC issue as a starting point for the deep convection issue.

**Comment by scollis** [comment]
Hey @wgustafson as a deep convection guy I love the idea of a version of LASSO with a wider domain to look at precipitating systems. 
This would allow us to go to a slightly courser resolution (100-500m) and really use some extra observations like scanning radar (Both DOE and NOAA) as well as platforms like GOES16..

I would love to look at a full blown assimilation project in ARM (which LASSO is, to a degree, for shallow Cu). 

One issue is I just don't know if the SGP is the place to do this.. Scales are massive (CONUS) at SGP. Old TWP sites (Darwin, Manus) might be more appropriate (LASSO-Hector... that would be FUN) but there may be some programatic issues with this.. Another though is the upcoming (2021) TRACER AMF deployment near Houston where we get a nice trade flow. 

In a nutshell: LASSO Deep would be deeply cool but is the SGP the right place? We would need to show how we can do better that HRRR..

(Or perhaps the best thing we can do is nest in HRRR.. hmmm)...

**Comment by davidturner53** [comment]
While the HRRR does pretty well with deep convection, it also doesn't always get it right either.  The primary reason it fails is that the mesoscale environment is wrong (but this isn't the only reason).  If ARM targets deep convection for the next version of LASSO, it will have to get a good set of measurements at the model boundaries.  It is possible that the AERIs, MWRs, and DLIDs at the 4 current profiling EFs would be sufficient if they were moved back out to the former BF facilities -- however, the profiles from these remote sensors are primarily limited to the lowest 2-3 km.  Will the lack of observations in the free troposphere be a problem?  

I know we tried this experiment before in the late 1990s / early 2000s, but the AERI retrievals (in particular) are much different (improved) now.  Perhaps an alternative: let's revisit the dataset collected from spring 1998 to summer 2003 (when the BFs had AERIs and 404 MHz wind profilers), and derive better thermodynamic profiles from the AERIs and perform the analysis.  The only hiccup I see in this suggestion is that the operational modeling systems of that era, which we would use as part of the variational analysis scheme, are much poorer than today's operational models so there would need to be some work done to determine how to best derive the forcing dataset used to drive the LES...

**Comment by wgustafson** [comment]
Closing this issue on GitHub. Folks interested in contributing further thoughts should post a comment in the LASSO category on <https://discourse.arm.gov/c/lasso/lasso-general/11>.
